<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Kabwes-MBP.attlocal.net" failures="0" tests="5" name="Default test" time="1.497" errors="0" timestamp="2025-02-01T12:28:59 CST">
  <testcase classname="testBase.BaseClass" name="@BeforeClass setup" time="0.544">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;String.equalsIgnoreCase(String)&amp;quot; because &amp;quot;os&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "String.equalsIgnoreCase(String)" because "os" is null
at testBase.BaseClass.setup(BaseClass.java:46)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- @BeforeClass setup -->
  <testcase classname="testBase.BaseClass" name="@AfterClass tearDown" time="-1.738434539637E9">
    <skipped/>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase classname="testcases.TC03_LoginDDT" name="verify_LoginData" time="0.0">
    <skipped/>
  </testcase> <!-- verify_LoginData -->
  <testcase classname="testcases.TC03_LoginDDT" name="verify_LoginData" time="0.0">
    <skipped/>
  </testcase> <!-- verify_LoginData -->
  <testcase classname="testcases.TC03_LoginDDT" name="verify_LoginData" time="0.0">
    <skipped/>
  </testcase> <!-- verify_LoginData -->
  <testcase classname="testcases.TC03_LoginDDT" name="verify_LoginData" time="0.0">
    <skipped/>
  </testcase> <!-- verify_LoginData -->
  <testcase classname="testcases.TC03_LoginDDT" name="verify_LoginData" time="0.0">
    <skipped/>
  </testcase> <!-- verify_LoginData -->
</testsuite> <!-- Default test -->
